modules/ROOT/pages/ch2-custom-alerts.adoc

```diff
@@ -1,136 +1,26 @@
-= Configuring Custom Notifications and Application Alerts
+= Configuring Application Alerts
 
 *THIS IS BASED ON CH 1.0 CONTENT*
 
-*Last update* 21-Sep-21
+*Last update* 11-Mar-22
 
-_Notifications_ are messages that appear in Anypoint Runtime Manager to provide status about your apps.
-_Alerts_ enable you to send emails based on your application's notifications.
-
-== Notifications 
-
-You access notifications from the bell icon on the applications pages in Runtime Manager. 
-When you have new notifications, the bell turns red:
-
-image::alerts-notification-icon.png[Bell notification icon on the Applications page]
-
-You can create _custom notifications_ to give you visibility into business-related events in your application.
-You create custom notifications using Anypoint Connector for CloudHub (CloudHub Connector).
-
-For example, you can create notifications that appear when:
-
-** Your application is unable to connect to a remote service.
-** An error that requires human intervention occurs, such as a problem with data mapping.
-** You want to create a summary of what happened in your application, such as the number of orders processed.
-
-=== Automatic Notifications
-
-*THIS CONTENT IS FROM CH 1.0*
-
-For applications deployed to CloudHub 2.0, some events automatically produce a notification:
-
-[%header,cols="30a,70a"]
-|===
-|Event |Message
-| Worker will restart | A worker `<ip>` has become unresponsive. Restarting the worker.
-| Worker has restarted | Restart SUCCEEDED for worker `<ip>`
-| Worker restart failed | Restart FAILED for worker `<ip>`
-| Maximum Restarts attempted for a worker| Maximum number of restart attempts reached
-| Worker needs restarting, but restarts are disabled | A worker `<ip>` has become unresponsive. Restarts are disabled.
-|===
-
-=== Notification Retention
-
-Anypoint Platform retains notifications up to the following limits, whichever occurs first:
-
-* 30 days
-* Maximum of 1000 notifications per application
-+
-If the number of notifications exceeds the maximum,
-CloudHub 2.0 deletes the oldest notification.
-+
-Each notification can contain up to 10,000 characters.
-
-
-== Alerts
-
-You can create standard alerts in Runtime Manager for applications deployed to CloudHub 2.0.
+_Alerts_ indicate whether an app is behaving as expected. You can create alerts in Runtime Manager for applications deployed to CloudHub 2.0.
 See https://docs.mulesoft.com/runtime-manager/alerts-on-runtime-manager[Alerts^] (Link out from Beta docs).
 
-If you create custom notifications, you can create custom alerts to send emails when 
-those notifications occur.
-
 === Rate Limits on Alerts
```

Comment:
I'm getting a section title out of sequence here.

Reply:
```suggestion
== Rate Limits on Alerts
```